packages/cli/__tests__/zosjobs/__unit__/submit/Submit.shared.handler.unit.test.ts

```diff
@@ -585,7 +612,7 @@ describe("submit shared handler", () => {
 
             expect(error).toBeUndefined();
             expect(SubmitJobs.submitJclString).toHaveBeenCalledTimes(1);
-            expect(copy.response.console.log).toHaveBeenCalledTimes(4);
+            // expect(copy.response.console.log).toHaveBeenCalledTimes(4);
```

Comment:
Should this line of the test be restored?

Reply:
yes sorry